tests/performance/test_kernel.py

```diff
@@ -47,7 +47,7 @@ def setUp(self) -> None:
 
 class TestSquaredExponentialKernel(TestKernel):
     """
-    Tests related to the SquaredExponentialKernel defined in kernel.py
+    Tests related to the SquaredExponentialKernel defined in `kernel.py`.
```

Comment:
These don't get rendered so it doesn't *really* matter, but for consistency please could they be in double back ticks?